### PR TITLE
split out answer key

### PR DESCRIPTION
**Pull request by elimillera**
I made a change in breakout one wording where we weren't doing a function for them but otherwise the only changes were to the code chunks.

### CODE REVIEW

File: breakout_4/breakout_4_walkthrough.Rmd [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+-   Once again, **group** the data by treatment group (ARM)
+-   **Count** the number of subjects within each age group \*AGEGR1)
+
+*Note: There are multiple ways to solve this one! You'll find with R that quite often, there's multiple ways to solve the same problem. If your organization doesn't have a preferred convention, do what feels most intuitive. And if something feels like it could be more straightforward - it's worth looking into because someone may already made a better approach.*
+
+```{r sum_agegr}
+## Using summarize
+dm %>% 
+  group_by(___, ___) %>% 
+  summarize(___, .groups='___') %>%
```
Comment (mstackhouse) [inline breakout_4/breakout_4_walkthrough.Rmd]: leave the `.groups`  answer here filled out - that's a bit wonky.

**Review by mstackhouse — COMMENTED** [review]
I'm good with the documents here. I want to change the names though. For the suffixes, how about:
- answers
- helper
- blank

**Comment by elimillera** [comment]
@mstackhouse Files names updated

**Review by mstackhouse — APPROVED** [review]
All good.

**Pull request merged**